wp/plugin/lib/analysis.ml

```diff
@@ -138,6 +138,19 @@ let sp (arch : Arch.t) : (Comp.comparator * Comp.comparator) option =
            memory for the x86_64 architecture.\n%!";
     None
 
+
```

Comment:
Wait, why do we have a `lib` sub-folder in our `plugin` folder? Shouldn't this be in `wp/lib`?

Reply:
Nevermind.